lib/Doctrine/Migrations/Metadata/AvailableMigration.php

```diff
@@ -7,6 +7,10 @@
 use Doctrine\Migrations\AbstractMigration;
 use Doctrine\Migrations\Version\Version;
 
+/**
+ * The class AvailableMigration an available migration.
```

Comment:
Looks like it misses a word, which I guess is `represents`, in which case  I would avoid this tautology.
```suggestion
 *  Available migrations may or may not be already executed
```